### DISCUSSION TITLE
Cancelling the manual saving process for multiple tabs?

### DISCUSSION THREAD

**Discussion by touwys** [gildas-lormeau/SingleFile#1575] [Q&A]
When saving multiple tabs manually, it appears to be impossible to cancel, at any stage during the Save process, the processing of the remaining tabs. Should the user wish to cancel the process, they are currently obliged to cycle through all of the remaining explorer windows and cancel each one in turn. Repeating this for large numbers of tabs, is bothersome.

**Comment by gildas-lormeau** [comment]
Open the context menu, select "SingleFile > View pending saves" and click on the "Cancel all" button. Alternatively, you can display this page via the side panel in the browser.

**Reply by touwys** [reply]
**> Open the context menu, select "SingleFile > View pending saves" and click on the "Cancel all" button.** 

Which context menu, specifically, are you referring to, and where is it located? The Explorer Window currently saving, is open on top of all others, and thus prevents access to context menus other than its own.

**> Alternatively, you can display this page via the side panel in the browser.**

I am uncertain what you mean by this. Which page? This one? How does it effect access to the said context menu. Can  you, please, elaborate?

Thank you.

**Reply by gildas-lormeau** [reply]
What browser are you using?

**Comment by touwys** [comment]
Thank you!
----------------------------------------

21 Sep 2024 15:23:01 Gildas ***@***.***>:

**Comment by touwys** [comment]
Please respond?

Thank you.

**Comment by touwys** [comment]
> What browser are you using?

Firefox 130.
----------------------------------------

24 Sep 2024 17:13:18 Gildas ***@***.***>:

**Reply by gildas-lormeau** [reply]
Thank you, you have to right-click on the page or the icon to open the context menu. See the screenshot below (cf. "View pending saves..." entry) for example.

<img width="470" alt="image" src="https://github.com/user-attachments/assets/6986f6ac-20b5-4dcc-bcc6-f97c41cd7eaf">

**Reply by touwys** [reply]
Thank you, I've noticed it with the most recent Single File update. However, unless I'm error, the SingleFile menu (i.e. right-click to access it) remains inaccessible while the multiple tabs saving process is ongoing (i.e. the Explorer dialogue to save the tab keeps popping as neccessary, urging user intervention to save)

**Reply by gildas-lormeau** [reply]
Unfortunately, it's impossible for the extension to close an opened dialog from a technical point of view. The only way to circumvent this issue is to disable the opening of the "save as" dialog when saving a page.

**Comment by touwys** [comment]
Thank you very much, that explains it.
----------------------------------------

27 Sep 2024 10:50:24 Gildas ***@***.***>:
